nilearn/plotting/tests/test_img_plotting/test_plot_glass_brain.py

```diff
@@ -114,3 +114,8 @@ def test_plot_glass_brain_with_completely_masked_img(display_mode):
     img = Nifti1Image(np.zeros((10, 20, 30)), np.eye(4))
     plot_glass_brain(img, display_mode=display_mode)
     plt.close()
+
+
+def test_plot_glass_brain_vmin_vmax(mni_3d_img):
+    """Smoke tests for plot_glass_brain beaing passed vmin and vmax."""
```

Comment:
```suggestion
    """Smoke tests for plot_glass_brain beaing passed vmin and vmax."""
```
being + breaking = beaing ? 😉